rskj-core/src/main/java/org/ethereum/rpc/TypeConverter.java

```diff
@@ -82,11 +108,11 @@ public static String toJsonHex(byte[] x) {
     }
 
     public static String toJsonHex(Coin x) {
-        return x != null ? x.asBigInteger().toString() : "" ;
+        return x != null ? x.asBigInteger().toString() : "";
```

Comment:
I believe this should `0x0` in case `x == null`.

Reply:
should we also prepend "0x" as BigInteger.toString() returns hex without "0x" ?

Reply:
Updated